### ISSUE TITLE
Avoiding overfitting of pretrained models

### ISSUE THREAD

**Issue by shin2875** [shin2875/cnn-pest-classification#7]
![densenet](https://github.com/shin2875/cnn-pest-classification/assets/127823226/37e234a9-cfb9-4fd7-8056-1a290fed653d)

Epochs seems to be ended before training is completed. How can I achieve higher accuracy without overfitting?

**Comment by shin2875** [comment]
You may add dropout layers or early stopping to halt the training of model.
